lib/ReactViews/BadgeBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { TextSpan } from "../Styled/Text";
+const Box = require("../Styled/Box").default;
+
+interface IProps {
+  label: string;
+  badge?: number;
+  children: React.ReactNode;
+}
+const BadgeBar: React.FC<IProps> = (props: IProps) => {
+  return (
+    <Box paddedHorizontally={3} justifySpaceBetween>
+      <Box verticalCenter>
+        <TextSpan textLight uppercase>
+          {props.label}
+        </TextSpan>
+        {props.badge && (
+          <TextSpan
+            textLight
+            uppercase
+            css={`
+              padding-left: 5px;
+            `}
+          >
+            ({props.badge})
+          </TextSpan>
+        )}
```

Comment:
```suggestion
        <TextSpan textLight uppercase>
          {props.label} {props.badge ? `(${props.badge})` : null}
        </TextSpan>
```

Last night I pushed a small fix for badge bar when having longer texts, just saw that I posted a commit link not branch link https://github.com/zoran995/terriajs-i18n/blob/workbench-ts/lib/ReactViews/BadgeBar.tsx

Reply:
many thanks @zoran995

Reply:
thanks @zoran995, I have also added the nowrap to ensure that it remains on one-line. One thing I might investigate tommorrow is to add ellipses and a title so that we might be able to fail a bit more gracefully when dealing with extra-long translation strings